quiche/src/h3/mod.rs

```diff
@@ -422,29 +422,86 @@ pub enum Error {
     VersionFallback,
 }
 
+/// Error codes as defined in https://www.rfc-editor.org/rfc/rfc9114.html#http-error-codes.
+pub enum Http3ErrorWireCode {
+    /// No error. This is used when the connection or stream needs to be closed, but there is no error
+    /// to signal.
+    H3NoError = 0x0100,
```

Comment:
Why the leading zero on this and others?

Reply:
I copy-pasted them from the RFC:

> H3_NO_ERROR (0x0100):
No error. This is used when the connection or stream needs to be closed, but there is no error to signal.